frontend/src/features/chat/components/composer/SlashCommandDropdown.tsx

```
@@ -31,17 +36,147 @@ const SlashCommandDropdown: React.FC<SlashCommandDropdownProps> = ({
+        break;
+
+      case "ArrowDown":
+        e.preventDefault();
+        if (onNavigateDown) {
+          onNavigateDown();
+        }
+        break;
+
+      case "ArrowLeft":
+        e.preventDefault();
+        const currentCategoryIndex = categories.indexOf(selectedCategory);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)